src/control/cmd/daos/container.go

```diff
@@ -351,6 +353,73 @@ func (cmd *containerCreateCmd) Execute(_ []string) (err error) {
 	return nil
 }
 
+type containerSerializeCmd struct {
+	daosCmd
+
+	Source string `long:"src" short:"s" description:"container to be serialized" required:"1"`
+	Output string `long:"output-path" short:"o" description:"path to output serialized HDF5 files" required:"0"`
```

Comment:
@daltonbohning yeah I wasn't sure...because the other utilities currently do not use positional arguments. I thought maybe that would be more confusing and easier to update all interfaces at once? Not really sure. I don't think that the output dir should be positional though (since it is optional), maybe just src.

Reply:
I'm not sure, but I guess it would be easier to update all datamover tools at once

Reply:
@daltonbohning I am updating it to use the positional arguments